### PR TITLE
Added Taylor_series_Horner_Rule in cpp,java,README.md

### LINKED ISSUE #1129: Taylor series in c++, java and README.md

### PR DESCRIPTION
**Pull request by Anshuvats**
Fixes #1129 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Implementation of Taylor_series using Horner Rule in Java,cpp and README.md files.

#### Languages Used:
- C++
- JAVA

#### Files Added:
- Taylor_series_Repo/README.md 
- Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp 
- Taylor_series_Repo/Taylor_Series_Horner_Rule.java

### CODE REVIEW

File: Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    static double s; // declare static variable
+    if (num == 0)   // Base case
+    {
+        return s;
+    }
+    s = 1 + x * (s / num); // formula of horner's rule
+    return exponential (x, num - 1); // recursive case
+}
+// main method
```
Comment (MastersAbh) [inline Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp]: A blank line before comment

File: Taylor_series_Repo/Taylor_Series_Horner_Rule.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Taylor series using Horner Rule in Java
+
+import java.util.Scanner;
+public class Test {
```
Comment (MastersAbh) [inline Taylor_series_Repo/Taylor_Series_Horner_Rule.java]: A blank line here

Comment (MastersAbh) [reply Taylor_series_Repo/Taylor_Series_Horner_Rule.java]: before class

File: Taylor_series_Repo/Taylor_Series_Horner_Rule.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Taylor series using Horner Rule in Java
+
+import java.util.Scanner;
+public class Test {
+    static double s; // Declare static variable
+    public static void main(String[] args) {
```
Comment (MastersAbh) [inline Taylor_series_Repo/Taylor_Series_Horner_Rule.java]: A blank line here before main function

**Comment by Anshuvats** [comment]
@MastersAbh I have updated these files. Kindly review this.

File: Taylor_series_Repo/Taylor_Series_Horner_Rule.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+public class Test {
+    static double s; // Declare static variable
+    
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int x = sc.nextInt();
+        int num = sc.nextInt();
+        double result = exponential(x, num);
+        System.out.println(+result);
```
Comment (MastersAbh) [inline Taylor_series_Repo/Taylor_Series_Horner_Rule.java]: Please remove + before result.

File: Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    return exponential (x, num - 1); // recursive case
+}
+
+// main method
+int main ()
+{
+    int x, num;
+    cin >> x >> num;
+    double result = exponential (x, num);
+    cout << result << endl; //x^num
```
Comment (MastersAbh) [inline Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp]: space after //

**Comment by Anshuvats** [comment]
Hey @MastersAbh, I have updated this. Please review this.

**Comment by Anshuvats** [comment]
@marwahmanbir @jainaman224, I have followed up with the changes Request.
Can you please check?

**Comment by MastersAbh** [comment]
@marwahmanbir you can have a look.

**Pull request merged**